components/custom_status/custom_status_modal.tsx

```diff
@@ -189,6 +189,10 @@ const CustomStatusModal: React.FC<Props> = (props: Props) => {
         </div>
     );
 
+    const showSuggestions = (currentCustomStatus && currentCustomStatus.emoji) ?
+        (!isStatusSet || (currentCustomStatus.emoji === emoji && text && currentCustomStatus.text === text)) :
+        !isStatusSet;
+
```

Comment:
couldn't this be simpified by using nullish coalescent?

```suggestion
    const showSuggestions = !isStatusSet || (currentCustomStatus?.emoji === emoji && text && currentCustomStatus?.text === text);
        
```